### PR TITLE
RHCLOUD-2441 - Row Item Just Download With Button

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

**Comment by tahmidefaz** [comment]
image

**Comment by tahmidefaz** [comment]
@karelhala This kinda looks crappy, and the code somewhat feels sketchy. Any suggestions on how to improve here?

**Comment by karelhala** [comment]
@tahmidefaz button in primary variant looks really off in the table. It would be better to use `plain` variant and instead of text use either `ExportIcon` or `FileDownloadIcon` from `@patternfly/react-icons` package.

File: src/Utilities/overviewRows.js [hunk 1]
```diff
@@ -35,6 +37,9 @@ export const rowMapper = (title, appName, version, selectedRows = []) => ({
         {
             title: <Badge>{ version }</Badge>,
             value: version
+        },
+        {
+            title: <Button onClick={ () => {downloadFile(appName, version);} }>Download JSON</Button>
```
Comment (karelhala) [inline src/Utilities/overviewRows.js]: No need to wrap the function in brackets.
```JSX
title: <Button onClick={ () => downloadFile(appName, version) }>Download JSON</Button>
```

**Comment by tahmidefaz** [comment]
How does this look: 
image

**Pull request merged**